extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -259,11 +259,11 @@ export class RemoteQueriesInterfaceManager {
   }
 
   /**
-   * Builds up a list of analysis results, in a data structure tailored to the view.
-   * @param analysisResults The results of a specific analysis.
+   * Builds up a list of analysis summaries, in a data structure tailored to the view.
+   * @param analysisResults The summary of a specific analysis.
```

Comment:
Very minor: Would it make sense for this param to be called `analysisSummaries` too? 🔍